src/core/ui/about.cpp

```diff
@@ -109,7 +108,7 @@ QString AboutDialog::tabAbout()
         .arg(QLatin1String("http://www.gnu.org/licenses/old-licenses/gpl-2.0.html"));
     str += QLatin1String("<br><br>");
 
-    str += tr("Copyright &copy; 2009-2013, Artem 'DOOMer' Galichkin");
+    str += tr("Copyright &copy; 2009-2025, LXQt team");
```

Comment:
2009 "LXQt team" didn't exist, first release was 2014
. Better keep Doomer and adding 
`    str += tr("Copyright &copy; 2014-2025, LXQt team");`

Reply:
Doomer is mentioned in the "special thanks" section. He has played his role.

Reply:
We shouldn't simlpy remove copyright info, especially when we didn't exist at that time.

Reply:
The old copyright info belonged to the old app. Since this isn't that app anymore, keeping it was wrong.

Reply:
> The old copyright info belonged to the old app

Copyright applies to codes, not apps. Most likely there are still a large amount of codes from Doomer's screengrab in this project. Doomer's copyright remains for those codes unless they had a copyright transfer agreement with us.

As usual, I am not a lawyer and the comment above is only my personal opinion.

Reply:
So, what should we do here, in your opinion?

Reply:
And please note that the original codes may not be recognizable anymore (especially, in the near future).

Reply:
Maybe something like:

``` suggestion
    str += tr("Copyright &copy; 2009-2013, Artem 'DOOMer' Galichkin");
    str += tr("Copyright &copy; 2013-2025, LXQt team");
```

I think I've seen that in other projects, but not sure what the super proper way would be either, if there is one...

Reply:
@Chiitoo 
Thanks a lot! This is the best suggestion. Will make it so.

Reply:
Done.

Reply:
> And please note that the original codes may not be recognizable anymore (especially, in the near future).

Do you mean original codes will mostly be replaced soon? Then, Doomer's copyright no longer exists in this project at that time and the statement can be removed.

> Maybe something like:
> 
> I think I've seen that in other projects, but not sure what the super proper way would be either, if there is one...

Yes, this is okay, thank you!

Reply:
> Do you mean original codes will mostly be replaced soon?

Let me put it this way: There are programs (like libfm-qt) whose core structures don't need any change for a long time. Features can be added to them and their issues can be fixed without changing those structures. ScreenGrab is not one of them.

Anyway, @Chiitoo's suggestion solved the problem.